src/emc/usr_intf/gmoccapy/gmoccapy.py

```diff
@@ -3224,6 +3224,13 @@ def on_key_event(self, widget, event, signal):
         # tooledit page is active, so keys must go through
         if self.widgets.ntb_preview.get_current_page() == 2:
             return
+            
+        if (self.widgets.ntb_preview.get_current_page() == 4 and
+            keyname in ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9',
+            'comma', 'period', 'BackSpace', 'Return']):
+            #user Tab, pass numbers through
+            return False
+            
```

Comment:
Code style only ;-)
```suggestion
            
        # user Tab, pass numbers through
        if self.widgets.ntb_preview.get_current_page() == 4 and
            keyname in ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9',
            'comma', 'period', 'BackSpace', 'Return']:
            return False
```

Reply:
Should 'comma', 'period', 'BackSpace', 'Return' not possibly be a constant of some class such that typos/API changes would be found at compile time? Google could not find it for me. I like the code style change.

Reply:
These constants are defined and derived from https://gitlab.gnome.org/GNOME/gtk/-/blob/main/gdk/gdkkeysyms.h

>The names are the same as those in the gdk/gdkkeysyms.h header file but without the leading “GDK_KEY_”.

\[https://lazka.github.io/pgi-docs/#Gdk-3.0/functions.html#Gdk.keyval_name ]

Reply:
Thank you for that pointer. Stared at it for a while but nothing surfaced that I could suggest to make this any better. Also checked https://lazka.github.io/pgi-docs/#Gtk-3.0/enums.html and https://lazka.github.io/pgi-docs/#Gtk-3.constants.html .